manifests/p/ppy/osu/2024.1224.1/ppy.osu.locale.zh-CN.yaml

```
@@ -18,6 +18,7 @@ CopyrightUrl: https://osu.ppy.sh/legal/en/Copyright
 ShortDescription: 节奏跃然指上
 Description: 一款免费的节奏游戏。节奏跃然指上。
 Tags:
+  - 音游
```

```suggestion
- 音游
```
Sorry, automation error.